mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterAkkaImpl.java

```diff
@@ -118,6 +119,16 @@ public CompletableFuture<List<TaskExecutorID>> getBusyTaskExecutors(Map<String,
             .thenApply(l -> l.getTaskExecutors());
     }
 
+    @Override
+    public CompletableFuture<List<TaskExecutorID>> getDisabledTaskExecutors(Map<String, String> attributes) {
+        return Patterns.ask(
+                resourceClusterManagerActor,
```

Comment:
Can you add an akka handler for this endpoint in `ResourceClustersManagerActor` as well?

`.match(GetDisabledTaskExecutorsRequest.class, req -> getRCActor(req.getClusterID()).forward(req, context()))`